### ISSUE TITLE
✅ ♿ 🐞 Bug Report: Posts widget links are not compliant with accessibility standards - [ED-11654]

### ISSUE THREAD

**Issue by theavier** [elementor/elementor#8087]
**What problem is your feature request going to solve? Please describe.**
In trying to get elementor-built wordpress site fully wcag-supported Im having issues with how links are contructed for the Posts-widget. In one Post-item I can land with a minimum of 2-3 links which lead to the same destination. 
According to wcag 2.4.4 we should avoid duplicate links 
https://webaim.org/standards/wcag/checklist#sc2.4.4

**Describe the solution you'd like**
It would be great if there was an alternative to make each post a single link. 
So you can click image, read more, or text and but its still all one link.

**Describe alternatives you've considered**
Not using Posts-widget for presenting information, but this seem like a simple solution. 

**Additional context**
In sweden there is a law that requires all goverment website to support wcag AA.

**Comment by anyssaki** [comment]
The main reason I'm not using Elementor is the lack of ADA compliance with WCAG 2.1 and Section 508 standards.  Most businesses and state agencies are required to be ADA compliant (AA level minimum).  If the theme builder supported ADA compliance, this would be an extremely valuable tool for developers, designers, and clients.  Unfortunately, until this feature is added, I won't be using Elementor.

**Comment by tiskraba** [comment]
+1

**Comment by tomas-eklund** [comment]
I agree. I came here to request the same thing. 

I'm trying to pitch WordPress/Elementor Pro for a school website remake initiated by the need to follow WCAG AA according to [European law](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2016.327.01.0001.01.ENG&toc=OJ:L:2016:327:TOC), but my initial research shows that Elementor is very far from WCAG compliant. The Posts widget has the problems mentioned by @theavier, which could quite easily be remedied. Other widgets have similar or other problems.

Could we please ask for some accessibility-related improvements of Elementor in the near future?

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.16! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
